src/main/java/bio/terra/cli/command/config/List.java

```diff
@@ -1,27 +1,49 @@
 package bio.terra.cli.command.config;
 
-import bio.terra.cli.context.GlobalContext;
-import java.util.concurrent.Callable;
```

Comment:
This looks like a "good" use of static imports to me, but I'm interested to see if @wchamber agrees:

go/java-practices/imports#static-good